bundles/org.openhab.binding.freeathomesystem/src/main/java/org/openhab/binding/freeathomesystem/internal/FreeAtHomeSystemBindingConstants.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.freeathomesystem.internal;
+
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link FreeAtHomeSystemBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Andras Uhrin - Initial contribution
+ */
+@NonNullByDefault
+public class FreeAtHomeSystemBindingConstants {
+
+    public static final String CONFIG_DESCRIPTION_URI_THING_PREFIX = "thing-type";
+
+    public static final String BINDING_ID = "freeathomesystem";
+
+    // List of all Thing Type UIDs
+    public static final String BRIDGE_TYPE_ID = "gateway";
+    public static final String FREEATHOMEDEVICE_TYPE_ID = "free-at-home-device";
```

Comment:
Since this is used for generating thing type IDs, perhaps it could be shortened to:
```suggestion
    public static final String FREEATHOMEDEVICE_TYPE_ID = "device";
```

UID would then become `freeathomesystem:device` rather than `freeathomesystem:free-at-home-device`.